src/ros_ibeo_lux.cpp

```diff
@@ -97,10 +85,16 @@ int main(int argc, char **argv)
   ros::Publisher object_contour_points_pub = n.advertise<visualization_msgs::Marker>("as_tx/object_contour_points", 1);
 
   ros::Publisher scan_data_pub, object_data_pub, vehicle_state_pub, error_warn_pub;
-  ros::Publisher fusion_scan_2204_pub, fusion_scan_2205_pub, fusion_object_2225_pub, fusion_object_2280_pub, fusion_img_2403_pub, fusion_vehicle_2806_pub, fusion_vehicle_2807_pub;
-
-  //LUX Sensor Only
-  if(!is_fusion)
+  ros::Publisher fusion_scan_2204_pub,
```

Comment:
This looks really weird to me. Maybe tab the wrapped ones in to match the top variable name?

Reply:
The ROS style guide doesn't specify here but the ROS2 guide does. Since this formatting clean-up is a precursor to ROS2 conversion and there is no current standard, my recommendation would be to go with the ROS2 style. See https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#examples.